client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -126,8 +126,12 @@ export class GoogleAnalyticsForm extends Component {
 		const wooCommerceActive = wooCommercePlugin ? wooCommercePlugin.active : false;
 
 		const nudgeTitle = siteIsJetpack
-			? translate( 'Enable Google Analytics by upgrading to Jetpack Premium' )
-			: translate( 'Enable Google Analytics by upgrading to the Business plan' );
+			? translate(
+					'Connect your site to Google Analytics in seconds with Jetpack Premium or Professional'
+			  )
+			: translate(
+					'Connect your site to Google Analytics in seconds with Jetpack Premium or Professional'
```

Comment:
We should have only changed the first line for Jetpack, the second line impact WordPress.com plans